docs/src/user/manual/data-upload/update-data.md

```diff
@@ -0,0 +1,96 @@
+---
+title: IDS SAWPS
+summary: The SANBI Wildlife Protection System is a platform to track the population levels of endangered wildlife.
+    - Jeremy Prior
+    - Ketan Bamniya
+date: 09-11-2023
+some_url: https://github.com/kartoza/sawps/
+copyright: Copyright 2023, SANBI
+contact: PROJECT_CONTACT
+license: This program is free software; you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version.
+context_id: BYbnmQvaLxzHsfjUtsaAFn
```

Comment:
This needs to be unique, you can't copy the context ID from another file (this was copied from reports.md). There is a script to generate new uuids (`create-uuid.py`)
```suggestion
context_id: 86E2FetEjDbkA6t7ET8efs
```